coordinator/server/client_api.go

```diff
@@ -39,6 +39,9 @@ type StatusResp struct {
 }
 
 type ManifestSignatureResp struct {
+	// The manifest signature - signed by the root ECDSA key. Changes upon each call due to a random component.
+	// example: MEYCIQCmkqOP0Jf1v5ZR0vUYNnMxmy8j9aYR3Zdemuz8EXNQ4gIhAMk6MCg00Rowilui/66tHrkETMmkPmOktMKXQqv6NmnN
```

Comment:
```suggestion
	// example: MEYCIQCmkqOP0Jf1v5ZR0vUYNnMxmy8j9aYR3Zdemuz8EXNQ4gIhAMk6MCg00Rowilui/66tHrkETMmkPmOktMKXQqv6NmnN
	// swagger:strfmt byte
```
OpenAPI workflow does not work for pull request from forks.

You can test it locally using the following:
```shell
swagger generate spec -m --compact --exclude-deps -o ./swagger.json
swagger validate ./swagger.json
```